crates/rust-analyzer/src/lsp/to_proto.rs

```diff
@@ -1786,12 +1800,24 @@ pub(crate) mod command {
 
     pub(crate) fn debug_single(runnable: &lsp_ext::Runnable) -> lsp_types::Command {
         lsp_types::Command {
-            title: "Debug".into(),
+            title: "⚙\u{fe0e} Debug".into(),
             command: "rust-analyzer.debugSingle".into(),
             arguments: Some(vec![to_value(runnable).unwrap()]),
         }
     }
 
+    pub(crate) fn update_single(
+        runnable: &lsp_ext::Runnable,
+        label: &Option<SmolStr>,
+    ) -> Option<lsp_types::Command> {
+        let runnable = make_update_runnable(runnable, label)?;
+        Some(lsp_types::Command {
+            title: format!("↺\u{fe0e} Update Tests ({})", label.as_ref()?),
+            command: "rust-analyzer.updateSingle".into(),
```

Comment:
Why do we need a new command for this? That requires this change to be implemented by each editor where as before we were merely using the env variables which would work out of the box for any client.

Reply:
If this is solely for the prompt that asks the user whether you want to update, we should make that VSCode specific in the sense that the vscode extension should check for the env vars in the runnable payload and act accordingly instead that way we do not need to expose a client implemented command here